raiden_contracts/deploy/__main__.py

```diff
@@ -68,7 +70,7 @@ def __init__(
         gas_limit: int,
         gas_price: int=1,
         wait: int=10,
-        contracts_version: Optional[str]=None,
+        contracts_version: Optional[PlainVersion]=None,
```

Comment:
Are you sure you are running flake8 on the codebase? There should be a space between `=` and `None`.

Reply:
Yes Travis runs `flake8`, but I'm not sure if the `flake8` configuration is the same as in `raiden`.

Reply:
`pylint` tells me about it.  https://github.com/raiden-network/raiden-contracts/issues/642